util/custom-response/src/main/java/com/brother/customresponse/payload/BaseDto.java

```diff
@@ -0,0 +1,14 @@
+package com.brother.customresponse.payload;
+
+import lombok.Data;
+
+import java.time.LocalDateTime;
+
+@Data
```

Comment:
Please use only required annotations.
Example: @getter, https://github.com/Setter and etc.

Reply:
> Please use only required annotations. Example: @getter, @Setter and etc.
Done